python/py-sacremoses/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        alvations sacremoses 99102ce
+version             0.0.35
+revision            0
+name                py-sacremoses
+
+license             GPL-2.1
+maintainers         nomaintainer
+platforms           darwin
+
+description         Python port of Moses tokenizer, truecaser and normalizer
+
+long_description    ${description}
```

Comment:
Generally, a long description should be, well, longer. I think setting it to `${description}` is an antipattern except in the case of things that are very very obvious.

Reply:
This is a cut-and-paste of the `README.md` so that's what we've got.